.github/scripts/image.sh

```diff
@@ -45,7 +38,16 @@ if [[ -n "${DOCKER_USER}" && -n "${DOCKER_PASSWORD}" ]]; then
     ${DOCKER} login -u ${DOCKER_USER} -p ${DOCKER_PASSWORD} ${DOCKER_REGISTRY}
     ${DOCKER} tag ${PROJECT_NAME}:${PROJECT_TAG} ${DOCKER_REGISTRY}/${DOCKER_ORG}/${PROJECT_NAME}:${PROJECT_TAG}
     ${DOCKER} push ${DOCKER_REGISTRY}/${DOCKER_ORG}/${PROJECT_NAME}:${PROJECT_TAG}
-    if ${PUSH_LATEST}; then
+
+    # PUSH_LATEST environment variable is exported only in release.yml
+    # Only when an actual release tag (for e.g. 2.1.0) is pushed, we push the :latest.
+    # :latest represents the latest released version of the software.
+    # We do not push :latest when main or other non-release tags are pushed.
+    if [ -z "$PUSH_LATEST" ]
+    then
```

Comment:
I'm used to see this on one line, but I guess does not really matter

```suggestion
    if [ -z "$PUSH_LATEST" ]; then
```

Also, I'd do the following just to be ready for the safety bash flags https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

```suggestion
    if [ -z "${PUSH_LATEST:-}" ]; then
```